apps/builder/app/builder/features/sidebar-left/panels/pages/custom-metadata.tsx

```diff
@@ -0,0 +1,173 @@
+import {
+  Button,
+  Grid,
+  InputErrorsTooltip,
+  InputField,
+  Label,
+  SmallIconButton,
+  Text,
+  theme,
+} from "@webstudio-is/design-system";
+import { DeleteIcon, PlusIcon } from "@webstudio-is/icons";
+import { useId } from "react";
+
+type Meta = {
+  property: string;
+  content: string;
+};
+
+type CustomMetadataProps = {
+  customMetas: Meta[];
+  onChange: (value: Meta[]) => void;
+};
+
+const PropertyContent = (props: {
+  property: string;
+  content: string;
+  onDelete: () => void;
+  onChange: (property: string, content: string) => void;
+}) => {
+  const propertyId = useId();
+  const contentId = useId();
+
+  return (
+    <Grid
+      gap={2}
+      css={{
+        gridTemplateColumns: `${theme.spacing[18]} 1fr 19px`,
+        gridTemplateAreas: `
+      "property property-input button"
+      "content content-input button"
+    `,
+      }}
+      align={"center"}
+    >
+      <Label htmlFor={propertyId} css={{ gridArea: "property" }}>
+        Property
+      </Label>
+      <InputErrorsTooltip errors={undefined}>
+        <InputField
+          css={{ gridArea: "property-input" }}
+          tabIndex={1}
+          id={propertyId}
+          property="path"
+          value={props.property}
+          onChange={(event) => {
+            props.onChange(event.target.value, props.content);
+          }}
+        />
+      </InputErrorsTooltip>
+      <Label htmlFor={contentId} css={{ gridArea: "content" }}>
+        Content
+      </Label>
+      <InputErrorsTooltip errors={undefined}>
```

Comment:
Why undefined?

Reply:
Have no idea will be some errors, or not. Like does duplicates are allowed etc. Decided to leave it here